packages/enterprise-search/src/api/enterprise/api.ts

```diff
@@ -70,6 +70,31 @@ export default class API {
     }, options)
   }
 
+  async getStorage (this: That, params?: T.GetStorageRequest, options?: TransportRequestOptions): Promise<T.GetStorageResponse> {
+    return await this.transport.request<T.GetStorageResponse>({
+      method: 'GET',
+      path: '/api/ent/v1/internal/storage'
```

Comment:
🤔 it seems like all of the paths in the [swagger file](https://github.com/elastic/ent-search/blob/main/swagger/v1/enterprise-search.json) used for this API are `/internal/`

Reply:
> should we be including any internal paths in a client?

No.

> 🤔 it seems like all of the paths in the [swagger file](https://github.com/elastic/ent-search/blob/main/swagger/v1/enterprise-search.json) used for this API are /internal/

That sounds right to me. Public facing APIs are being built as we speak for new Enterprise Search public-facing APIs, in [this other swagger file](https://github.com/elastic/ent-search/blob/main/swagger/enterprise-search.json) (not released to users yet)

Reply:
Agreed with Rich, let's skip internal APIs.

Reply:
I will update this to use [this other enterprise-search.json](https://github.com/elastic/ent-search/blob/main/swagger/enterprise-search.json) spec file.

Also going to re-generate to 8.5 since 8.6 has been pushed to January.